### PR TITLE
README.md: newline at end of file

### PR DESCRIPTION
**Pull request by stappersg**
The is missing newline character was discovered
while preparing another patch.

### CODE REVIEW

**Comment by sigoden** [comment]
We don't care.

**Comment by stappersg** [comment]
On Tue, Dec 24, 2024 at 03:24:24AM -0800, sigoden wrote:
> We don't care.

OK.


For what it is worth:

* Initial change was done with github.com webinterface
* I was surprised about an extra change I didn't make
* Was unable to see what the change was
* After a `git clone` to Linux machine and `git show`
  noticed the missing new line character
* Removed the 0x0A with a hexeditor to get a clean #46
* Made git branch "newline"
* Opened README.md in neoviom and saved the README.md
* Got a newline at end-of-file, this time without surprise
* Created this merge request


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by sigoden** [comment]
Sorry, I apologize if my remarks were too direct or impolite.

Adding/removing a newline at the end of an md file doesn't make much sense.

**Comment by stappersg** [comment]
On Tue, Dec 24, 2024 at 03:51:07AM -0800, sigoden wrote:
> Sorry, I apologize if my remarks were too direct or impolite.

No worries,  we are good.

 
> Adding/removing a newline at the end of an md file doesn't make much sense.

Yes, we, you and I, agree on that.  That there is software that deem a
newline at end-of-file important, is something we deal with.


Groeten
Geert Stappers

P.S.
Merry Christmas
-- 
Silence is hard to parse
